src/modules/widgets/reducer.ts

```diff
@@ -126,6 +127,24 @@ const widgetsReducer = (
           },
         },
       };
+    case SAVE_CLONED_WIDGET:
+      return {
+        ...state,
+        items: {
+          ...state.items,
+          [action.payload.id]: {
+            ...action.payload.widgetItem,
+            widget: {
+              ...action.payload.widgetSettings,
+              id: action.payload.id,
+              position: {
+                ...action.payload.widgetSettings.position,
+                y: 999_999_999,
```

Comment:
For some reason, `Infinity` option doesn't work for adding the widget at the bottom, so I've added a big Integer
https://github.com/STRML/react-grid-layout/blob/master/test/examples/6-dynamic-add-remove.jsx#L77

Reply:
`Infinity` option is not supported correctly by `react-grid-layout`, it is better to use big Integer as You suggested, I just moved it to constant, that can be easily replaced etc.